site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+
+2.  **Give ***businesses*** a well-lit path for efficiently loading third-party resources.**
+
+    We would like to propose new standards that encourage browsers to more intelligently make trade-offs between how first-party and third-party resources are loaded in the name of a better loading experience for users. Later, we'll highlight some of these proposals, such as lazy-loading third-party embeds by default or applying different resource prioritisation to third-party resources we don't believe to be critical to the initial content users may care about most. These are just a small number of the ideas we're evaluating in this space and would love to collaborate with both web performance experts and the broader community on shaping this work.
+
+    We would similarly like to address such problems in frameworks and Content Management Systems (CMS) where more appropriate. Projects such as [Aurora](http://web.dev/aurora) and [WordPress Performance Team](https://github.com/WordPress/performance) have taught us the importance of baked-in defaults that resolve known loading issues. Defaults baked into frameworks and CMS guide businesses along a well-lit path. They can also be helpful to the user agent (for example, Chrome) as hints allowing it to apply heuristics to optimize page load and CWV. Such hints can help the user agent decide when and how specific third-parties should load in the page life cycle. (For example, the [Next.js script component](https://web.dev/script-component/#the-next.js-script-component) has a baked-in default to load third-party scripts after the page becomes interactive.)
+
+3.  **Give ***third-parties*** incentives to reduce their performance impact via better transparency efforts.**
+
+    Third-party developers currently lack the visibility required to understand the impact of their scripts on sites at large. We plan to address this issue and equip these providers with tools to analyze their impact and compare it with other products in the market that provide the same value. We also want to help them use the data to diagnose what causes the impact so that they can mitigate it upstream. We will have to call out all third-parties, including those authored by Google, to succeed.
+
+## Goals
+
+We will know we have contributed significantly to reducing the impact of third-parties on CWV when we have achieved the following goals.
+
+-   50% decrease in partner origins who struggle to hit Core Web Vitals because of third-parties.
+-   50% of third-parties identified as most problematic for Core Web Vitals have a well-lit path solution.
+
+## Challenges
+
+An effort of this magnitude is not without challenges. Some of the key challenges we have to consider are.
+
+-   Third-parties are a cross-cutting problem involving ads, analytics, tag management, utilities, and many others. Each area requires consideration of a unique set of requirements and trade-offs. For example,
```

Comment:
Should this end in "for example," with a dangling comma? Or an unfinished sentence? If the latter, my bad for missing it in the draft.